packages/opc-styles/README.md

```diff
@@ -0,0 +1,14 @@
+# Welcome to OPC Style Library 👋
+
+This packages contains the all common styles which are important to get
+started with any component which includes global variables, colors, common mixin etc.
+
+![Version](https://img.shields.io/badge/version-0.0.1-blue.svg?cacheSeconds=2592000)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/1-Platform/op-components/graphs/commit-activity)
+
```

Comment:
Do we have some documentation for the styles? It would be great if we could add the doc directly to the readme, or maybe in `docs` folder, and link it here.\
It should be open and public.

[Although I wouldn't suggest linking any google docs.]

Reply:
yeah, the readme file is updated.